base/applications/taskmgr/perfpage.c

```diff
@@ -8,59 +8,47 @@
 #include "precomp.h"
 #include <shlwapi.h>
 
-extern BOOL bInMenuLoop;        /* Tells us if we are in the menu loop - from taskmgr.c */
-
-TM_GRAPH_CONTROL PerformancePageCpuUsageHistoryGraph;
-TM_GRAPH_CONTROL PerformancePageMemUsageHistoryGraph;
-
-HWND hPerformancePage;                /* Performance Property Page */
-static HWND hCpuUsageGraph;                  /* CPU Usage Graph */
-static HWND hMemUsageGraph;                  /* MEM Usage Graph */
-HWND hPerformancePageCpuUsageHistoryGraph;           /* CPU Usage History Graph */
-HWND hPerformancePageMemUsageHistoryGraph;           /* Memory Usage History Graph */
-static HWND hTotalsFrame;                    /* Totals Frame */
-static HWND hCommitChargeFrame;              /* Commit Charge Frame */
-static HWND hKernelMemoryFrame;              /* Kernel Memory Frame */
-static HWND hPhysicalMemoryFrame;            /* Physical Memory Frame */
-static HWND hCpuUsageFrame;
-static HWND hMemUsageFrame;
-static HWND hCpuUsageHistoryFrame;
-static HWND hMemUsageHistoryFrame;
-static HWND hCommitChargeTotalEdit;          /* Commit Charge Total Edit Control */
-static HWND hCommitChargeLimitEdit;          /* Commit Charge Limit Edit Control */
-static HWND hCommitChargePeakEdit;           /* Commit Charge Peak Edit Control */
-static HWND hKernelMemoryTotalEdit;          /* Kernel Memory Total Edit Control */
-static HWND hKernelMemoryPagedEdit;          /* Kernel Memory Paged Edit Control */
-static HWND hKernelMemoryNonPagedEdit;       /* Kernel Memory NonPaged Edit Control */
-static HWND hPhysicalMemoryTotalEdit;        /* Physical Memory Total Edit Control */
-static HWND hPhysicalMemoryAvailableEdit;    /* Physical Memory Available Edit Control */
-static HWND hPhysicalMemorySystemCacheEdit;  /* Physical Memory System Cache Edit Control */
-static HWND hTotalsHandleCountEdit;          /* Total Handles Edit Control */
-static HWND hTotalsProcessCountEdit;         /* Total Processes Edit Control */
-static HWND hTotalsThreadCountEdit;          /* Total Threads Edit Control */
-
-#ifdef RUN_PERF_PAGE
-static HANDLE hPerformanceThread = NULL;
-static DWORD  dwPerformanceThread;
-#endif
+#define MAX_CPU_PER_LINE    16 // TODO: Make this selectable in submenu.
+
+// typedef struct _CPU_GRAPH CPU_GRAPH, *PCPU_GRAPH;
+static ULONG CpuTotalPanes = 0;
+static PTM_GRAPH_CONTROL CpuUsageHistoryGraphs = NULL; /* CPU Usage History Graphs Array */
+
+static HWND hCpuUsageHistoryGraph; /* CPU Usage History Graph */
+static TM_GRAPH_CONTROL CpuUsageHistoryGraph; /* CPU Usage History Graph template control */
+static RECT rcCpuGraphArea; /* Rectangle area for CPU graphs */
+
+static HWND hMemUsageHistoryGraph; /* Memory Usage History Graph */
+static TM_GRAPH_CONTROL MemUsageHistoryGraph;
+
+static HWND hCpuUsageGraph; /* CPU Usage Graph */
+static HWND hMemUsageGraph; /* MEM Usage Graph */
```

Comment:
```suggestion
static HWND hMemUsageHistoryGraph;
static TM_GRAPH_CONTROL MemUsageHistoryGraph;

static HWND hCpuUsageGraph;
static HWND hMemUsageGraph;
```
That's the exact type of comments that do not add any value at all. But in a few years someone will start to align the comments at the exact same column. A year later someone will add dots. A year later the dots will be removed. Then  somebody will think: Hey let's change from /* */ to // to save a few bytes. Let's end it once and for all time by stripping comments that have absolutely no value. This is pure cargo-cult-documentation!